### ISSUE TITLE
Social.localUser.Authenticate((bool success) does not return True or False

### ISSUE THREAD

**Issue by raheelzz** [playgameservices/play-games-plugin-for-unity#2499]
Having issues integrating the google play services in unity using android manifest and gradle build.

I have more than one sdk integrated into the project as well. example: Applovin, Appsflyer, IronSource

I have also made Game Services and did the setup for GooglePlayGames Android Setup and it resolved successfully

I added myself as a tester and made sure the google play service recognizes the fingerprint from my local keystore SHA1 certificate

The error I am getting is:

ERROR: Exception launching token request: java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment

and this error happens when I run :

Social.localUser.Authenticate((bool success) 

but this method does not return a True or False

**Comment by heishi1989** [comment]
I have the same problem

I use Unity 2018.3.0f2 and GooglePlayGamesPlugin-0.9.62

Is there any way to solve this problem?

**Comment by raheelzz** [comment]
Solved it by not making a custom gradle build, but still a gradle build. Unity's gradle compiler implements whatever is needed through an automated process. This can also result in bloating the file size because all .aar files get packaged (implementation) in the apk.

Ideal situation would be to know which APKs to use and include those only in the custom gradle template.

Not sure if this would work for anyone else with similar issue. The reason I went this way was because the documentation for IronSource mentioned it (using Custom Gradle) but not necessarily the correct solution
